integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/client/ElasticsearchClientFactoryImplIT.java

```diff
@@ -150,6 +156,38 @@ public void simple_http() {
 		}
 	}
 
+	@Test
+	public void simple_httpClientConfigurer() throws Exception {
+		String payload = "{ \"foo\": \"bar\" }";
+		String statusMessage = "StatusMessage";
+		String responseBody = "{ \"foo\": \"bar\" }";
+		wireMockRule1.stubFor( post( urlPathMatching( "/myIndex/myType" ) )
+				.withRequestBody( equalToJson( payload ) )
+				.andMatching( httpProtocol() )
+				.willReturn( elasticsearchResponse().withStatus( 200 )
+						.withStatusMessage( statusMessage )
+						.withBody( responseBody ) ) );
+
+		HttpResponseInterceptor responseInterceptor = spy( HttpResponseInterceptor.class );
```

Comment:
If you're going to start using Mockito somewhere, please use the MockitoRule in the test class. See for example `org.hibernate.search.backend.elasticsearch.client.impl.ElasticsearchClientUtilsGetElasticsearchVersionTest#mockito`.